commit bdb4b61e569b3cc745f674d44bf088ad98df13bc
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Tue Apr 24 14:28:33 2018 +0100

    BCM270X_DT: Add sdio_overclock parameter
    
    The Arasan MMC interface is used on some RPis to drive the SDIO
    link to the WiFi controller. The downstream bcm2835-mmc driver,
    like the bcm2835-sdhost driver, can be over- (or under-) clocked.
    Add a common parameter - sdio_overclock - to all DTBs to control it.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/bcm2708-rpi.dtsi b/arch/arm/boot/dts/bcm2708-rpi.dtsi
index d105d4e208f2..fe2694a1cbdc 100644
--- a/arch/arm/boot/dts/bcm2708-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2708-rpi.dtsi
@@ -115,6 +115,7 @@
 		sd_force_pio = <&sdhost>,"brcm,force-pio?";
 		sd_pio_limit = <&sdhost>,"brcm,pio-limit:0";
 		sd_debug     = <&sdhost>,"brcm,debug";
+		sdio_overclock = <&mmc>,"brcm,overclock-50:0";
 		axiperf      = <&axiperf>,"status";
 	};
 };
diff --git a/arch/arm/boot/dts/overlays/README b/arch/arm/boot/dts/overlays/README
index ca6995142322..81e61c0d6bc2 100644
--- a/arch/arm/boot/dts/overlays/README
+++ b/arch/arm/boot/dts/overlays/README
@@ -139,6 +139,9 @@ Params:
 
         sd_debug                Enable debug output from SD driver (default off)
 
+        sdio_overclock          Clock (in MHz) to use when the MMC framework
+                                requests 50MHz for the SDIO/WiFi interface.
+
         tx_lpi_timer            Set the delay in microseconds between going idle
                                 and entering the low power state (default 600).
                                 Requires EEE to be enabled - see "eee".
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1235_bdb4b61e569b3cc745f674d44bf088ad98df13bc.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1235_bdb4b61e569b3cc745f674d44bf088ad98df13bc.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
